File: apps/ios/Barns/BarnsApp/Domain/Entities/CareGuide.swift
import Foundation

struct CareGuide: Equatable, Identifiable, Sendable {
    let id: String
    let title: String
    let categoryId: String
    let summary: String
    let steps: [String]
    let frequency: String
    let cautions: [String]
    let imageUrl: URL?
}

File: apps/android/app/src/main/java/com/barns/app/domain/model/CareGuide.kt
package com.barns.app.domain.model

data class CareGuide(
    val id: String,
    val title: String,
    val categoryId: String,
    val summary: String,
    val steps: List<String>,
    val frequency: String,
    val cautions: List<String>,
    val imageUrl: String?,
)
